### PR TITLE
Add an option to disable page reloading between tests

### PR DESCRIPTION
**Pull request by AndreyBelym**

### CODE REVIEW

**Comment by AndreyBelym** [comment]
@testcafe-build-bot retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot retest

File: src/cli/argument-parser.js [hunk 1]
```diff
@@ -108,6 +108,7 @@ export default class CLIArgumentParser {
             .option('--hostname <name>', 'specify the hostname')
             .option('--proxy <host>', 'specify the host of the proxy server')
             .option('--proxy-bypass <rules>', 'specify a comma-separated list of rules that define URLs accessed bypassing the proxy server')
+            .option('--disable-reloading', 'disable page reloading between tests')
```
Comment (VasilyStrelyaev) [inline src/cli/argument-parser.js]: .option('--no-page-reload', 'disable page reloads between tests')

File: src/runner/index.js [hunk 2]
```diff
@@ -212,8 +212,9 @@ export default class Runner extends EventEmitter {
         return this;
     }
 
-    run ({ skipJsErrors, quarantineMode, debugMode, selectorTimeout, assertionTimeout, pageLoadTimeout, speed = 1, debugOnFail } = {}) {
+    run ({ skipJsErrors, disableReloading, quarantineMode, debugMode, selectorTimeout, assertionTimeout, pageLoadTimeout, speed = 1, debugOnFail } = {}) {
         this.opts.skipJsErrors     = !!skipJsErrors;
+        this.opts.disableReloading = !!disableReloading;
```
Comment (VasilyStrelyaev) [inline src/runner/index.js]: `disablePageReloads` everywhere

File: src/runner/index.js [hunk 3]
```diff
@@ -212,9 +212,9 @@ export default class Runner extends EventEmitter {
         return this;
     }
 
-    run ({ skipJsErrors, disableReloading, quarantineMode, debugMode, selectorTimeout, assertionTimeout, pageLoadTimeout, speed = 1, debugOnFail } = {}) {
+    run ({ skipJsErrors, noPageReload, quarantineMode, debugMode, selectorTimeout, assertionTimeout, pageLoadTimeout, speed = 1, debugOnFail } = {}) {
         this.opts.skipJsErrors     = !!skipJsErrors;
-        this.opts.disableReloading = !!disableReloading;
+        this.opts.noPageReload     = !!noPageReload;
```
Comment (VasilyStrelyaev) [inline src/runner/index.js]: can't we name it `disablePageReloads` in JS API?

File: src/runner/index.js [hunk 4]
```diff
@@ -212,8 +212,9 @@ export default class Runner extends EventEmitter {
         return this;
     }
 
-    run ({ skipJsErrors, quarantineMode, debugMode, selectorTimeout, assertionTimeout, pageLoadTimeout, speed = 1, debugOnFail } = {}) {
+    run ({ skipJsErrors, noPageReload, quarantineMode, debugMode, selectorTimeout, assertionTimeout, pageLoadTimeout, speed = 1, debugOnFail } = {}) {
         this.opts.skipJsErrors     = !!skipJsErrors;
+        this.opts.noPageReload     = !!noPageReload;
```
Comment (VasilyStrelyaev) [inline src/runner/index.js]: can't we name it `disablePageReloads` in JS API?

File: src/compiler/test-file/formats/raw.js [hunk 5]
```diff
@@ -32,6 +32,9 @@ export default class RawTestFileCompiler extends TestFileCompilerBase {
         if (src.authCredentials)
             dest.httpAuth(src.authCredentials);
 
+        if (src.disableReloading !== void 0)
```
Comment (miherlosev) [inline src/compiler/test-file/formats/raw.js]: `src.disablePageReloads`

**Comment by miherlosev** [comment]
ping @VasilyStrelyaev

**Pull request merged**